ocaml/src/asd_server.ml

```diff
@@ -394,7 +394,7 @@ module Net_fd = struct
        in
        let writer buffer offset length =
          let write_from_source = Lwt_ssl.write_bytes socket buffer in
-         Lwt_extra2._write_all write_from_source offset length
+         Lwt_extra2._write_all write_from_source (identifier fd_out) offset length
```

Comment:
This converts the fd to an int (and on the callee side back into an fd). 
It's not necessary and drops any type safety we have.